cmd/aro/update_ocp_versions.go

```diff
@@ -34,57 +39,111 @@ func getInstallerImageDigests(envKey string) (map[string]string, error) {
 	if !env.IsLocalDevelopmentMode() {
 		jsonData, err = base64.StdEncoding.DecodeString(string(jsonData))
 		if err != nil {
-			return nil, fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
+			return fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
 		}
 	}
 
-	if err = json.Unmarshal(jsonData, &installerImageDigests); err != nil {
-		return nil, fmt.Errorf("%s: Failed to parse JSON: %v", envKey, err)
+	if err = json.Unmarshal(jsonData, envData); err != nil {
+		return fmt.Errorf("%s: Failed to parse JSON: %v", envKey, err)
```

Comment:
:hear_no_evil: same scope-creep as above, make this generic error useful by including a reference to image digests, and use `%w`

Reply:
Same answer as above, though note it does print the environment variable name first.

So it's going to be one of:
```
INSTALLER_IMAGE_DIGESTS: Failed to parse JSON: ...

OPENSHIFT_VERSIONS: Failed to parse JSON: ...
```

I can change `%v` to `%w` although I'm not sure I understand the difference.

Reply:
Updated the "Refactoring" commit to change `%v` to `%w`.